resources/object_repo/installed_operators_obj.resource

```diff
@@ -6,8 +6,9 @@ Object Repository for Installed Operators Screen
 ${elemDbaasStatus}          //a[contains(.,"OpenShift Database Access Operator")]//ancestor::tr/td[contains(.,"Succeeded")]
 ${titleDbaasOperator}       //h1[contains(.,"OpenShift Database Access Operator") and contains(@class,"logo__name")]
 ${elemDbaasInvStatus}       //td[contains(.,"<<paname>>")]/parent::tr/td[contains(.,"SpecSynced, InventoryReady")]
+${titleDbaasInvertories}    //h1[contains(.,"DBaaSInventories")]
+${lnkProvAcc}               //a[contains(.,"OpenShift Database Access Operator")]//ancestor::tr/td/div/a[contains(.,"Provider Account")]
```

Comment:
Shall we update the xpath to - "//a[contains(.,"OpenShift Database Access Operator")]//ancestor::tr//a[contains(.,"Provider Account")]" to cut down the relative dependency.

Reply:
Thank you for the suggestion. Tried, tested and updated with new commit.